Lib/test/test_code_module.py

```diff
@@ -5,8 +5,7 @@
 from textwrap import dedent
 from contextlib import ExitStack
 from unittest import mock
-from test.support import import_helper
-
+from test.support import force_not_colorized_test_class, import_helper
```

Comment:
It is better to not mix a submodule import with other attributes import.

Reply:
Like this?
```suggestion
from test.support import force_not_colorized_test_class
from test.support import import_helper
```
Why is it better? Is this advice documented somewhere like the devguide or PEP 8?

Reply:
I am not sure that this case is explicitly documented, but PEP 8 recommends imports on separate lines, and the only exception is a from-import, but the example does not contain submodules. All existing code contains submodule import on separate line. So this is rather an unspoken rule.

Reply:
OK, I'll change it for consistency but don't really see the benefit, they're all just things to import.

(I think we should just automatically sort imports with isort/ruff, but that's another discussion :)